Updated Cortex XDR IOCs pack names - 20.6.0 (#7437)
demisto/content: Packs/XDR_iocs/Integrations/XDR_iocs/README.md

> Use the Cortex XDR - IOCs feed integration to sync indicators from XSOAR to XDR.

Reviewer: ```suggestion
Use the Cortex XDR - IOCs feed integration to sync indicators from Cortex XSOAR to Cortex XDR and back.
```

Cortex XDR is the world's first detection and response app that natively integrates network, endpoint and cloud data to stop sophisticated attacks.

> Use the Cortex XDR - IOCs feed integration to sync indicators between Cortex XSOAR and Cortex XDR. The integration will sync indicators according to the fetch interval defined. At each interval the integration will push new and modified indicators (selected according to the defined **Sync Query**) from Cortex XSOAR to Cortex XDR. Additionally, the integration will check if there are manual modifications of indicators on Cortex XDR and sync back to Cortex XSOAR. Once daily the integration will perform a *complete sync* which will also remove indicators, which have been deleted/expired in Cortex XSOAR, from Cortex XDR. 

Author: ```suggestion
Use the Cortex XDR - IOCs feed integration to sync indicators between Cortex XSOAR and Cortex XDR. The integration will sync indicators according to the defined fetch interval. At each interval, the integration will push new and modified indicators defined in the **Sync Query** from Cortex XSOAR to Cortex XDR. Additionally, the integration will check if there are manual modifications of indicators on Cortex XDR and sync back to Cortex XSOAR. Once per day, the integration will perform a *complete sync* which will also remove indicators that have been deleted/expired in Cortex XSOAR, from Cortex XDR. 
```
